docker/lnd/Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV GODEBUG netdns=cgo
 
 # Install dependencies and install/build lnd.
 RUN apk add --no-cache \
+    build-base \
```

Comment:
why `build-base` instead of `--update alpine-sdk` and `gcc`?  Not that I have a reason for one versus the other, i'm just curious what the reasoning is.

Reply:
no, probably your suggestion is better.